app/assets/javascripts/discourse/app/lib/webauthn.js

```diff
@@ -117,18 +118,33 @@ export function getWebauthnCredential(
 }
 
 export async function getPasskeyCredential(
-  challenge,
   errorCallback,
-  mediation = "optional"
+  mediation = "optional",
+  isFirefox = false
 ) {
   if (!isWebauthnSupported()) {
     return errorCallback(I18n.t("login.security_key_support_missing_error"));
   }
 
+  // we need to check isConditionalMediationAvailable for Firefox
+  // without it, Firefox will throw console errors
+  // We cannot do a general check because iOS Safari and Chrome in Selenium quietly support the feature
+  // but they do not support the PublicKeyCredential.isConditionalMediationAvailable() method
+  if (mediation === "conditional" && isFirefox && window.PublicKeyCredential) {
+    const isCMA =
+      // eslint-disable-next-line no-undef
```

Comment:
Is this `disable` needed? I see `PublicKeyCredential` is used in `confirm-session.gjs` w/o it

Reply:
It is indeed needed by eslint, the `publicKeyCredential` in `confirm-session.gjs` is a constant.

Reply:
Ah, true! So here I'd either add `/* global PublicKeyCredential */` on the first line of the file or stick to `window.PublicKeyCredential` that's used just above

Reply:
Ah yes, I cleaned this up a bit further. `isWebauthnSupported()` already checks that `PublicKeyCredential` is defined, so we don't need the additional check. And ended up using `/* global PublicKeyCredential */`, it's cleaner than peppering `window.`.

Thanks!